command/loadbot/execution.go

```diff
@@ -0,0 +1,219 @@
+package loadbot
+
+import (
+	"context"
+	"fmt"
+	"github.com/0xPolygon/polygon-sdk/helper/tests"
+	txpoolOp "github.com/0xPolygon/polygon-sdk/txpool/proto"
+	"github.com/0xPolygon/polygon-sdk/types"
+	"github.com/umbracle/go-web3"
+	"github.com/umbracle/go-web3/jsonrpc"
+	"google.golang.org/grpc"
+	empty "google.golang.org/protobuf/types/known/emptypb"
+	"math/big"
+	"sync"
+	"time"
+)
+
+// Configuration represents the loadbot run configuration.
+// It contains the required parameters to run the stress test.
+type Configuration struct {
+	TPS       uint64          // Number of transactions per second
+	Value     *big.Int        // The value sent in each transaction
+	Gas       uint64          // The transaction's gas
+	GasPrice  *big.Int        // The transaction's gas price
+	Accounts  []types.Address // All the accounts used by the bot to send transaction
+	RPCURLs   []string        // The JSON RPC endpoints, used to send transactions
+	ChainID   uint64
+	TxnToSend uint64 // The number of transaction to send
+	GRPCUrl   string // The gRPC url, used while verifying transactions to check if the TxPool is empty
+}
+
+type Metrics struct {
+	m         sync.Mutex
+	Total     uint64        // The total number of transactions processed
+	Failed    uint64        // The number of failed transactions
+	Duration  time.Duration // The execution time of the loadbot
+	TxnHashes []web3.Hash   // The hashes of the transactions sent
+}
+
+// createClients will create JSON RPC clients using the provided addresses in the CLI.
+// Each one of these clients will send transaction(s), each one after another.
+func (c *Configuration) createClients() ([]*jsonrpc.Client, error) {
+	var clients []*jsonrpc.Client
+
+	for _, url := range c.RPCURLs {
+		conn, err := jsonrpc.NewClient(url)
+		if err != nil {
+			return nil, fmt.Errorf("failed to connect to server: %v", err)
+		}
+		clients = append(clients, conn)
+	}
+	return clients, nil
+}
+
+// We create the transactions using createTransactionObjects before the loadbot send them.
```

Comment:
Just a nitpick, please avoid adding dots to end of code comments, it's not consistent with the rest of the code style we already have

Reply:
Removed in https://github.com/0xPolygon/polygon-sdk/pull/197/commits/45b67ce0874d8dbd2120ee3c409098b5a1f99c15